docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-planning-problem-con.adoc

```diff
@@ -0,0 +1,53 @@
+[id='optimizer-planning-problem-con']
+= What is a planning problem?
+
+
+A planning problem has an optimal goal, based on limited resources and under specific constraints. Optimal goals can be any number of things, such as:
```

Comment:
I'll only mention it once since it comes up a few other places, and since I think Emily already mentioned it. IBM SG is big on complete sentences before colons, without ending with "such as", "the following", etc. Can be resolved as easily as "such as the following examples:" I personally am not a huge fan and actually prefer what you've done, as an economy. But just sayin'. I've confirmed my stuff accordingly as well.

Reply:
Fixed in a couple of places, some other parts are still a WIP